src/features/rewards/siteBanner/index.tsx

```diff
@@ -287,20 +284,19 @@ export default class SiteBanner extends React.PureComponent<Props, State> {
                     </StyledNoticeWrapper>
                     : null
                 }
-                <StyledTitle isMobile={isMobile} isTwitterTip={(screenName && screenName !== '') ? true : false}>
+                <StyledTitle isTwitterTip={(screenName && screenName !== '') ? true : false}>
```

Comment:
I know this ternary was inline before, but can we cache it in to a variable this rendering? Would look a lot cleaner :)